### ISSUE TITLE
[4.0] jQuery used to open tab

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#23965]
Currently the code to link to a tab uses jquery. Not only do we want to get rid of jquery but the code currently doesn't work

### Steps to reproduce the issue
Go to  Plugins: Content - Page Break and click on "Show full description"

### Expected result
go to the description tab

### Actual result
Nothing except for this error in the console

```
joomla-tab.js?9cc7cf41ee2d6c979e5638cff870766e:276 Uncaught TypeError: Cannot read property 'click' of null
    at HTMLElement.showTab (joomla-tab.js?9cc7cf41ee2d6c979e5638cff870766e:276)
    at HTMLElement.connectedCallback (joomla-tab.js?9cc7cf41ee2d6c979e5638cff870766e:159)
    at joomla-tab.js?9cc7cf41ee2d6c979e5638cff870766e:2
    at joomla-tab.js?9cc7cf41ee2d6c979e5638cff870766e:401
```

**Comment by vijay9908** [comment]
Issue Confirmed
Can I work on this?

**Comment by vijay9908** [comment]
Can I know what alternative to jquery can be used?

**Comment by HLeithner** [comment]
vanilla es6 JS should be used.

@dgrammatiko may can help here?

**Comment by vijay9908** [comment]
can someone help me with location of text file from where contents are displayed earlier?

**Comment by dgrammatiko** [comment]
@vijay9908 please test #24098 

EDIT: my bad, I fixed the wrong issue in that PR

**Comment by dgrammatiko** [comment]
@brianteeman just replace the onclick with `document.querySelector('#tab-description').click()`

**Comment by ghost** [comment]
closed as having Pull Request #24098
